src/main/java/seedu/address/model/person/Date.java

```diff
@@ -32,47 +35,20 @@ public Date(LocalDateTime date) {
         value = date;
     }
 
-    @Override
-    public String toString() {
-        return value != LocalDateTime.MIN ? value.format(DateTimeFormatter.ofPattern("d/M/yyyy HHmm")) : "";
-    }
-
-    @Override
-    public boolean equals(Object other) {
-        return other == this // short circuit if same object
-                || (other instanceof Date // instanceof handles nulls
-                && value.equals(((Date) other).value)); // state check
-    }
-    @Override
-    public int hashCode() {
-        return value.hashCode();
-    }
-
     /**
-     * Validates the format and logical values of the provided date string.
-     *
-     * <p>This method checks if the given date string matches the expected format pattern of 'd/M/yyyy HHmm'
-     * (e.g., '2/12/2024 1800'),
-     * where 'd' is the day, 'M' is the month, 'yyyy' is the four-digit year, and 'HHmm' is the 24-hour time.
-     * It first ensures the date adheres to the general format and, if so,
-     * proceeds to validate the values for day, month, and year combinations, including special
-     * cases like leap years and months with fewer days.</p>
-     *
-     * <p>If the date format or any of the values are invalid,
-     * a {@code ParseException} is thrown with a detailed message indicating the issue.</p>
-     *
-     * @param date The date string to validate, in the format 'd/M/yyyy HHmm'.
-     * @throws ParseException if the date format does not match 'd/M/yyyy HHmm', or if the values are logically invalid,
-     *                        such as an incorrect day or an invalid leap year date.
+     * Returns true if a given string is a valid date.
```

Comment:
Maybe the javadocs can include information on what constitutes a valid date and also explain when an exception is thrown